### ISSUE TITLE
xmrig 2.6.1 stopped to build on ARMv7

### ISSUE THREAD

**Issue by Zenitur** [xmrig/xmrig#610]
```
[ 54%] Building CXX object CMakeFiles/xmrig.dir/src/workers/CpuThread.cpp.o
In file included from /home/pi/altcoins/xmr-stak/xmrig-2.6.1/src/workers/CpuThread.cpp:33:0:
/home/pi/altcoins/xmr-stak/xmrig-2.6.1/src/crypto/CryptoNight_arm.h:39:29: фатальная ошибка: crypto/c_keccak.h: Нет такого файла или каталога
 #include "crypto/c_keccak.h"
                             ^
компиляция прервана.
```

Hovewer 2.5.3 successfully builds.

**Comment by xmrig** [comment]
Already fixed in master branch.
Thank you.

**Comment by xmrig** [comment]
Fixed in v2.6.2.
